cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -111,10 +111,11 @@ func buildFlags(args *Args) []cli.Flag {
 			Usage:       "whether to disable blob cache manager",
 			Destination: &args.DisableCacheManager,
 		},
-		&cli.BoolFlag{
-			Name:        "enable-metrics",
-			Usage:       "whether to collect metrics",
-			Destination: &args.EnableMetrics,
+		&cli.StringFlag{
+			Name:        "metrics-address",
+			Value:       "",
+			Usage:       "metrics `ENDPOINT` does not listen by default",
```

Comment:
Maybe `Enable metrics server by setting to an address such as "localhost:8080", ":8080"`.

Reply:
Yes, It is more reasonable. Fixed.